.github/workflows/go.yml

```diff
@@ -1,31 +1,34 @@
 name: Build and Test
 on: [push, pull_request]
+permissions:
+  contents: read
+
 jobs:
   build:
     name: Go CI
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
     strategy:
       matrix:
-        go: [1.15, 1.16]
+        go: ['1.19', '1.20']
     steps:
       - name: Set up Go
-        uses: actions/setup-go@v2
+        uses: actions/setup-go@6edd4406fa81c3da01a34fa6f6343087c207a568 #v3.5.0
         with:
           go-version: ${{ matrix.go }}
       - name: Check out source
-        uses: actions/checkout@v2
+        uses: actions/checkout@8f4b7f84864484a7bf31766abe9204da3cbe65b3 #v3.5.0
       - name: Install build dependencies
         run: |
           sudo apt-get update
           sudo apt-get install -y beignet-dev nvidia-cuda-dev nvidia-cuda-toolkit ocl-icd-opencl-dev opencl-headers nvidia-opencl-dev
       - name: Install Linters
-        run: "curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin v1.39.0"
+        run: curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.52.2
       - name: Build OpenCL
         env:
-          GO111MODULE: "on"
+          CL_TARGET_OPENCL_VERSION: "220"
+
         run: go build -tags opencl ./...
+      - name: Lint
+        run: golangci-lint run
```

Comment:
`golangci-lint -c ./.golangci.yml run`

It was a problem for me with dex because I have a .golangci.json to tweak the IDE's golangci-lint config, and this vanilla `run` command preferred the json instead of the yml.